keymanagementscenarios.md

```diff
@@ -0,0 +1,105 @@
+Key management for container signing can be broadly categorized into three general use cases:
+- Key Setup/Signing (How are keys set up? How are keys accessed by Notary v2 client when signatures are generated?)
+- Trust store configuration (How do runtime environments determine which keys to trust?)
+- Revocation (How do key owners create/revoke keys? How do runtime environments get this information?)
+
```

Comment:
Should there be a definition section?  specifically I am looking for definitions of (or a pointer to the definition of) the root key, delegate keys, and subordinate keys as used in hthis doc, as well as a description of the key chaining concepts used in this doc.   it's really hard to follow the doc without knowing the meaning of these terms in the notary v2 context and how the chaining capabilities work.  (i.e. how many levels of chaining are supported, etc) These concepts are critical to get right as they are some of the areas where the notary v1 was most lacking.